app/Main.hs

```diff
@@ -116,9 +116,34 @@ main = do setLocaleEncoding utf8
               preloads = optPreload fullOpts
               postloads = optPostload fullOpts
               load = flip loadFiles
+              executionModeDoc = "(defmodule Introspect (doc execution-mode \"Returns the mode the Carp compiler is currently executing in.\"))"
```

Comment:
I'm not sure I understand why this is defined so early (and in such an unorthodox way...)

Please enlighten me :)

Reply:
I can make it inline in the call to `execStr` -- admittedly I had originally combined the doc declarations and var declarations, so I bound it to a var to help readability since the string was quite long!

Reply:
I inlined it now e0f34f9 hopefully that makes more sense!